server/repository/src/migrations/v1_01_01/mod.rs

```diff
@@ -0,0 +1,116 @@
+use super::{version::Version, Migration};
+use crate::StorageConnection;
+
+pub(crate) struct V1_01_01;
+
+impl Migration for V1_01_01 {
+    fn version(&self) -> Version {
+        Version::from_str("1.1.1")
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        use crate::migrations::sql;
+
+        // Indexes
+        sql!(
+            connection,
+            r#"
+            -- Foreign Keys (defined as REFERENCES)
+            CREATE INDEX "index_stocktake_line_item_id_fkey" ON "stocktake_line" ("item_id");
+            CREATE INDEX "index_stocktake_line_location_id_fkey" ON "stocktake_line" ("location_id");
+            CREATE INDEX "index_stocktake_line_stock_line_id_fkey" ON "stocktake_line" ("stock_line_id");
+            CREATE INDEX "index_stocktake_line_stocktake_id_fkey" ON "stocktake_line" ("stocktake_id");
+            CREATE INDEX "index_invoice_line_location_id_fkey" ON "invoice_line" ("location_id");
+            CREATE INDEX "index_invoice_line_stock_line_id_fkey" ON "invoice_line" ("stock_line_id");
+            CREATE INDEX "index_invoice_line_item_id_fkey" ON "invoice_line" ("item_id");
+            CREATE INDEX "index_invoice_line_invoice_id_fkey" ON "invoice_line" ("invoice_id");
+            CREATE INDEX "index_activity_log_store_id_fkey" ON "activity_log" ("store_id");
```

Comment:
not `record_id` as well? the usual lookup for the log is by the related record id

Reply:
🙏 f21937a1c98d3c62f7a771c86f32da0bdb1a2eec